src/java.base/linux/classes/jdk/internal/ffi/generated/socket/socket_address_h.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+// Generated by jextract
+
+package jdk.internal.ffi.generated.socket;
+
+@SuppressWarnings("restricted")
+public class socket_address_h {
+
+    socket_address_h() {
+        // Should not be called directly
+    }
+
+    private static final int AF_INET = (int)2L;
+    /**
+     * {@snippet lang=c :
+     * #define AF_INET 2
+     * }
+     */
+    public static int AF_INET() {
+        return AF_INET;
```

Comment:
We could inline the values here. Pro: smaller byte code; Con: more modifications of generated code.